release-notes/3.0/preview/api-diff/3.0 vs 2.2/standalone-packages/3.0.0-standalone-packages_System.Runtime.CompilerServices.Unsafe.md

```diff
@@ -0,0 +1,82 @@
+# System.Runtime.CompilerServices.Unsafe
+
+``` diff
+ {
+ {
+-    namespace System.Runtime.CompilerServices {
```

Comment:
Did we move `System.Runtime.CompilerServices.Unsafe`in-box? I wasn't aware of this. Is this expected? We still ship the OOB package, but I thought this was a "pure" OOB:
https://github.com/dotnet/corefx/blob/b893e604ce832ccf4e9cbcbdfe0b0ae2b23195bf/src/System.Runtime.CompilerServices.Unsafe/pkg/System.Runtime.CompilerServices.Unsafe.pkgproj#L19

https://github.com/dotnet/corefx/pull/25959

https://github.com/dotnet/corefx/issues/26145#issuecomment-355679947
https://github.com/dotnet/corefx/pull/28395

cc @ericstj, @jkotas

Reply:
We build the same binary to ship both inbox and OOB.  There still remains an `Internal.Runtime.CompilerServices.Unsafe` type in System.Private.CoreLib but we don't unify this type `System.Runtime.CompilerServices.Unsafe`.

Here is the record of this decision: https://github.com/dotnet/corefx/issues/33508

Reply:
@Anipik I don't think it's correct to represent this as a diff.  Something is wonky with your method for producing the .md